trades.py

```diff
@@ -674,7 +674,7 @@ def parse_user(  # pylint: disable=too-many-locals,too-many-statements
     creator: str,
     creator_trades_json: Dict[str, Any],
     mech_statistics: Dict[str, Any],
-) -> tuple[str, Dict[Any, Any]]:
+) -> tuple:
```

Comment:
Please import `Tuple` from `typing` and then:

```suggestion
) -> Tuple[str, Dict[Any, Any]]:
```